.travis.yml

```diff
@@ -21,7 +21,7 @@ install:
 - cd $PROJECT_DIR
 - travis_retry npm install grunt-cli
 - travis_retry npm install
-  cd ..
+- cd ..
```

Comment:
@agubler don't worry about the missing '-' just please don't let it happen again